testing/integration/logs_ingestion_test.go

```diff
@@ -128,7 +128,7 @@ func startMockES(t *testing.T) string {
 		uid,
 		clusterUUID,
 		nil,
-		time.Now().Add(time.Hour), 0, 0, 0, 100, 0))
+		time.Now().Add(time.Hour), 0, 0, 0, 0, 0))
```

Comment:
This causes the mock to accept all the events, instead of returning an error. The test that relied on the error has been updated.

Reply:
I ended up updating the code so whomever is creating the mock can pass the desired probabilities.